migrate-dao.js

```diff
@@ -90,26 +90,50 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
 
     const [founderAddresses, tokenDist, repDist] = [
       founders.map(({ address }) => address),
-      founders.map(({ tokens }) => web3.utils.toWei(tokens.toString())),
-      founders.map(({ reputation }) => web3.utils.toWei(reputation.toString()))
+      founders.map(({ tokens }) => web3.utils.toWei(tokens !== undefined ? tokens.toString() : '0')),
+      founders.map(({ reputation }) => web3.utils.toWei(reputation !== undefined ? reputation.toString() : '0'))
     ]
 
-    spinner.start('Creating a new organization...')
+    const foundersBatchSize = 40
+
+    let foundersInitCount = founderAddresses.length < foundersBatchSize ? founderAddresses.length : foundersBatchSize
     const forgeOrg = daoCreator.methods.forgeOrg(
       orgName,
       tokenName,
       tokenSymbol,
-      founderAddresses,
-      tokenDist,
-      repDist,
+      founderAddresses.slice(0, foundersInitCount),
+      tokenDist.slice(0, foundersInitCount),
+      repDist.slice(0, foundersInitCount),
       migrationParams.useUController === true ? UController : '0x0000000000000000000000000000000000000000',
       '0'
     )
 
-    const Avatar = await forgeOrg.call()
-    tx = await forgeOrg.send()
+    tx = await forgeOrg.send({
+      'gas': 6500000
+    })
+
+    const Avatar = tx.events.NewOrg.returnValues._avatar
+
     await logTx(tx, 'Created new organization.')
 
+    let foundersToAddCount = founderAddresses.length - foundersBatchSize
+    let i = 0
+    while (foundersToAddCount > 0) {
+      i++
+      spinner.start('Adding founders...')
+      let currentBatchCount = foundersToAddCount < foundersBatchSize ? foundersToAddCount : foundersBatchSize
+      tx = await daoCreator.methods.addFounders(
+        Avatar,
+        founderAddresses.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount),
+        tokenDist.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount),
+        repDist.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount)
+      ).send({
+        'gas': 6500000
```

Comment:
how about opt.gas ?

Reply:
I'm not sure if that's really necessary actually. I think I might just add it when trying to fix something but I see we already set a reasonable gas in the migrate.js script. I'll just remove it for now.